khmer/khmer_args.py

```diff
@@ -425,6 +425,16 @@ def calculate_graphsize(args, graphtype, multiplier=1.0):
 def create_nodegraph(args, ksize=None, multiplier=1.0, fp_rate=0.01):
     """Create and return a nodegraph."""
     args = _check_fp_rate(args, fp_rate)
+    
+    if hasattr(args, 'force'):
+        if not args.force:
+            if args.n_tables > 20:
+                print_error("\n** ERROR: khmer only supports number of tables <= 20.\n")
+                sys.exit(1)
+        else:
+            if args.n_tables > 20:
```

Comment:
It might be more clearly structured like so.

```
if args.n_tables > 20:
    if args.force:
        # warning
    else:
        # error + exit
```